tests/functions/test_collapse_levels.py

```diff
@@ -63,3 +67,87 @@ def test_collapse_levels_functionality_3level(
             "decorated-elephantAsDfr.i.p-rhino :'(AsDfdeadly__flamingo",
         ]
     )
+
+
+def mklbl(prefix, n):
```

Comment:
Could you help me understand what `mklbl` stands for? Perhaps adding the intent of the function into the docstring?

Reply:
nice catch! helps in creating a mutiindex dataframe. removing it, as it is a duplicate